src/NServiceBus.AcceptanceTests/Routing/When_publishing_to_scaled_out_subscribers_on_unicast_transports.cs

```diff
@@ -71,7 +71,8 @@ public class SubscriberA : EndpointConfigurationBuilder
         {
             public SubscriberA()
             {
-                EndpointSetup<DefaultServer>(c => { c.UnicastRouting().AddPublisher(PublisherEndpoint, typeof(MyEvent)); });
+                //EndpointSetup<DefaultServer>(c => { c.Routing().AddPublisher(PublisherEndpoint, typeof(MyEvent)); });
+                EndpointSetup<DefaultServer>(c => { c.UseTransport<MsmqTransport>().RegisterPublisherForType(PublisherEndpoint, typeof(MyEvent)); });
```

Comment:
Another use case would be if somebody is using non-generic `UseTransport` but honestly I am not sure what would be the use case for the non-generic version.